### ISSUE TITLE
Auto stacking for interval/timelapse photos?

### ISSUE THREAD

**Issue by anaxmedia** [photoprism/photoprism#1446]
I shoot a lot of timelapses and typically I use the auto stacking capability in Lightroom based on capture time and interval time and it works 95% of the time. I see there is stacking in PhotoPrism but is there any way to setup these photos that are part of a timelapse set to auto stack on import/index in PhotoPrism? Or the ability to exclude them from being indexed/imported with keywords/file names? They really cloud my albums.

**Comment by lastzero** [comment]
Stacking is enabled, but these timelapse photos are not recognized? What's the time difference between each shot?

**Comment by anaxmedia** [comment]
Yes that’s correct. Time difference is usually 5 seconds sometimes 10 seconds.

**Comment by lastzero** [comment]
That makes sense as PhotoPrism currently only stacks pictures taken within one second. They would also be stacked if they share the same image / document ID, but that's probably not the case as well. Sound like we need to add settings for the interval time, or implement manual stacking?

**Comment by lastzero** [comment]
Can you provide us with example files for testing? If you like also via email / link to hello@photoprism.app 👍

**Comment by anaxmedia** [comment]
> That makes sense as PhotoPrism currently only stacks pictures taken within one second. They would also be stacked if they share the same image / document ID, but that's probably not the case as well. Sound like we need to add settings for the interval time, or implement manual stacking?

That’s how Lightroom does it you can adjust the capture time interval and it increases/decreases the number of stacks. Here’s how they’ve implemented the functionality: https://helpx.adobe.com/lightroom-classic/help/grouping-photos-stacks.html

If you scroll all the way down to the bottom of the page it talks about their automatic stacking. I’m attaching 5 photos that are part of a timelapse for testing/examples. Let me know if you have any questions or need help testing anything! Thank you, you guys are doing a phenomenal job with this project. 

![20171231_082035_71835106](https://user-images.githubusercontent.com/7167565/126879710-1e343a66-e10e-4af2-8b83-6ee7d5046679.jpg)

![20171231_082040_623E3525](https://user-images.githubusercontent.com/7167565/126879651-60383c45-35ce-487b-999b-665f619eb125.jpg)

![20171231_082045_AF62DBC0](https://user-images.githubusercontent.com/7167565/126879833-fcb965db-df10-4b30-ac84-092e605f5064.jpg)

![20171231_082050_5441747B](https://user-images.githubusercontent.com/7167565/126879657-ccbb5c5d-1cf6-49b9-abff-baf502d3edd7.jpg)

**Comment by lastzero** [comment]
Thanks! We'll look into this as soon as possible. On vacation next week. Need to complete facial recognition as well, so it may take a few weeks.

**Comment by kitforbes** [comment]
Manual stacking would also be nice a option to have. I've got a few pictures of the same subject taken at random intervals over a minute or two from the same location. I'd like to group them into a single set if possible.
